[D1] D1 RR Documentation (#20922)
cloudflare/cloudflare-docs: src/content/docs/d1/configuration/data-location.mdx

:::caution
D1 location hints are not currently supported for South America (`sam`), Africa (`afr`), and the Middle East (`me`). D1 databases do not run in these locations.
:::

## Read replica locations

> D1 read replication allows you to create and distribute read-only copies of the primary database instance around the world. This reduces the query latency for users located far away from the primary database instance.

Reviewer: ```suggestion
With D1 read replication enabled we create and distribute read-only copies of the primary database instance around the world. This reduces the query latency for users located far away from the primary database instance.
```

Author: Reframed as "D1" doing the creating, rather than the user.

> When using D1 read replication through [Sessions API](/d1/features/read-replication/#how-d1-read-replication-works), D1 automatically creates a read replica in [every available region](/d1/configuration/data-location#available-location-hints), including the region where the primary database instance is located.

Reviewer: This is not factual. It's not using the Sessions API that creates the replicas, it's enabling replication that does it.

Using the Sessions API works just fine without replicas created.
